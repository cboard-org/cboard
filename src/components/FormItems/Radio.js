import React from 'react';
import MUIRadio from 'material-ui/Radio';
import { FormControlLabel } from 'material-ui/Form';

const Radio = props => <FormControlLabel control={<MUIRadio />} {...props} />;

export default Radio;
