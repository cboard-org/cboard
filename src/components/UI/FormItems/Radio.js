import React from 'react';
import MUIRadio from '@material-ui/core/Radio';
import FormControlLabel from '@material-ui/core/FormControlLabel';

const Radio = props => <FormControlLabel control={<MUIRadio />} {...props} />;

export default Radio;
