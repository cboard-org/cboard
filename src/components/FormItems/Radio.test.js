import React from 'react';
import { shallowMatchSnapshot } from '../../common/test_utils';
import Radio from './Radio.component';

describe('Radio tests', () => {
  test('default renderer', () => {
    shallowMatchSnapshot(<Radio />);
  });
});
