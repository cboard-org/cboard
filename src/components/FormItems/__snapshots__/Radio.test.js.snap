// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Radio tests default renderer 1`] = `
<WithStyles(FormControlLabel)
  control={<WithStyles(Radio) />}
/>
`;
